_summaries/misra2019mish_2.md

```diff
@@ -1,19 +1,34 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: misra2019mish_2
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: biofizzatreya
+score: 7
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+This paper attempts to fix the vanishing gradient problem of Relu activation functions by using, what they call Mish. Mish is defined as $$ x\cdot tanh\left(\text{ln}(1+e^x)\right) $$. The authors compare the effect of using different activation functions such as Relu, Swish and Mish. 
```

Comment:
I think it would be good to expand some of the motivations of Mish and why they're important here:
* non-monotonicity
* ability to preserve small negative weights: "fix the vanishing gradient problem"
* smooth profile: related to differentiability. Allows for smoother gradients.

Reply:
I think it would be useful to explain the "vanishing gradient" problem of ReLU by paraphrasing  "Dying ReLU ... is experienced through a gradient information loss caused by collapsing the negative inputs to zero"  which will further motivating Mish's "ability to preserve small negative weights".

Reply:
Since Mish is so closely related to Swish, it might be good to mention this relation and elaborate further when presenting the activation graphs.